clients/graphql-kotlin-client/src/main/kotlin/com/expediagroup/graphql/client/types/AutomaticPersistedQueriesSettings.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 Expedia, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.expediagroup.graphql.client.types
+
+data class AutomaticPersistedQueriesSettings(
+    val enabled: Boolean,
+    val version: Int
+)
+
+val defaultAutomaticPersistedQueriesSettings = AutomaticPersistedQueriesSettings(
+    enabled = false,
+    version = 1
```

Comment:
For now i would just expose the `enabled` setting, not the version, other APQ implementations does not allow specifying the version as of now there only exists 1.